ocs_ci/ocs/resources/mcg.py

```diff
@@ -1074,3 +1015,52 @@ def status(self):
             == get_default_bc["status"]["phase"]
             == STATUS_READY
         )
+
+    def create_nsfs_mcg_account(
+        self,
+        email,
+        name,
+        has_login,
+        s3_access,
+        allowed_buckets,
+        default_resource,
+        uid,
+        gid,
+        new_buckets_path,
+        nsfs_only,
+    ):
+        """
+        Create a new MCG account with the given parameters
+
+        Args:
+            email (str): Email address of the user
+            name (str): Name of the user
+            has_login (bool): Whether the user has a login
+            s3_access (bool): Whether the user has access to S3
+            allowed_buckets (str): Comma separated list of allowed buckets
+            default_resource (str): Default resource for the user
+            uid (str): UID of the user
+            gid (str): GID of the user
+            new_buckets_path (str): Path to the new buckets
+            nsfs_only (bool): Whether the user has access to NSFS only
+
+        Returns:
+            str: The name of the account created
+
+        """
+        params = {
+            "email": email,
+            "name": name,
+            "has_login": has_login,
+            "s3_access": s3_access,
+            "allowed_buckets": allowed_buckets,
+            "default_resource": default_resource,
+            "nsfs_account_config": {
+                "uid": uid,
+                "gid": gid,
+                "new_buckets_path": new_buckets_path,
+                "nsfs_only": nsfs_only,
+            },
+        }
+        logger.info(f"Creating MCG account with params: {params}")
+        return self.send_rpc_query("account_api", "create_account", params)
```

Comment:
Does it make sense to verify account creation and return s3 creds instead of the response?
Unlike handling in individual tests here: https://github.com/red-hat-storage/ocs-ci/pull/5393/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R4517